src/farcasterd/runtime.rs

```diff
@@ -2233,3 +2251,83 @@ pub fn launch(
         err
     })
 }
+#[derive(Default)]
+struct Syncers(HashMap<(Coin, Network), Syncer>);
```

Comment:
This should be `type Syncers = HashMap<...>` instead I think.

Reply:
Either you go back and implement some Deref trait or similar to avoid forwarding, or you keep the type definition and use a trait to implement custom methods on the HashMap, e.g

```rust
type Link = Option<Box<Node>>;

trait LinkMethods {
    fn cool_method(&self);
}

impl LinkMethods for Link {
    fn cool_method(&self) {
        // ...
    }
}
```